pallet/account-migration/src/lib.rs

```diff
@@ -248,9 +255,13 @@ pub mod pallet {
 			if threshold < 2 {
 				Self::migrate_inner(&from, &to)?;
 
-				Self::deposit_event(Event::MultisigMigrated { from, detail });
+				Self::deposit_event(Event::MultisigMigrated { from: from.clone(), detail });
 			} else {
-				<Multisigs<T>>::insert(from, detail);
+				<Multisigs<T>>::insert(&from, detail);
+			}
+
+			if let Some(to) = new_multisig_params {
+				Self::deposit_event(Event::NewMultisigParamsNoted { from, to });
```

Comment:
Should we deposit event in the case of `new_multisig_params` is `NONE`? Maybe we should.

Reply:
Nope